src/main/java/com/google/api/codegen/config/ResourceDescriptorConfig.java

```diff
@@ -218,7 +232,25 @@ private String getUnqualifiedTypeName() {
         ImmutableList.copyOf(descriptorConfigMap.values());
     for (Map.Entry<String, ResourceDescriptorConfig> entry : descriptorConfigMap.entrySet()) {
       ResourceDescriptorConfig childResource = entry.getValue();
+      System.out.println(
+          "DEL: Looking at entry "
```

Comment:
Are these intended, or are they the debug artifacts?

Reply:
Removed.